go.mod

```diff
@@ -162,7 +162,7 @@ require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/dimchansky/utfbom v1.1.1 // indirect
 	github.com/docker/libtrust v0.0.0-20150526203908-9cbd2a1374f4 // indirect
-	github.com/dustin/go-humanize v1.0.0 // indirect
+	github.com/dustin/go-humanize v1.0.0
```

Comment:
Related to the above; evidently we did not use this package before, so perhaps we should look if `docker/go-units` should be preferred for this.